tests/kernel/queue/prj.conf

```diff
@@ -1,3 +1,4 @@
 CONFIG_ZTEST=y
 CONFIG_IRQ_OFFLOAD=y
 CONFIG_TEST_USERSPACE=y
+CONFIG_MP_NUM_CPUS=1
```

Comment:
you need to explain, in detail, why you are disabling multi-core on this test. "kernel/queue requires single core" is not sufficient. especially since this passes on other arches.

Reply:
it's not needded after adding z_impl_z_test_1cpu_start. I will reverse it